packages/flutter/lib/src/material/reorderable_list.dart

```diff
@@ -87,6 +88,9 @@ class ReorderableListView extends StatefulWidget {
   /// List [children] can only drag along this [Axis].
   final Axis scrollDirection;
 
+  /// Controls scrolls and measures scroll progress.
```

Comment:
I would describe this a little more with regards to its use in a `ReorderableListView`. Use relevant info from [ScrollController](https://master-api.flutter.dev/flutter/widgets/ScrollController-class.html)'s docs, as well as describing common use cases for wanting to provide your own scroll controller.